android/java/AmpliApp/README.md

```diff
@@ -0,0 +1,24 @@
+# Ampli Android SDK (Java)
+An example app using the Ampli Android SDK
+
+# Usage
+
+### Setup the project
+You will need to do the following before running the app.
+1. Set your Amplitude API key in `src/main/java/com/example/ampliapp/App.java` file
+
```

Comment:
Please add `local.properties.example` with `#sdk.dir=/path/to/Android/sdk` and update README. See comment in Kotlin example.

Reply:
Added